src/components/EmptyListMessage.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components/macro';
+import iconFeather from '../assets/icon_feather.png';
+import { useNavigate } from 'react-router-dom';
+
+export default function EmptyListMessage() {
+  const navigate = useNavigate();
+
+  function handleClick(e) {
+    navigate('../form');
+  }
```

Comment:
Why do you specify the event here but don't use it? Additionally, I would suggest to use `event` instead of e.